data/filedefs/filetypes.HCL.conf

```diff
@@ -0,0 +1,40 @@
+# For complete documentation of this file, please see Geany's main documentation
+[styling=C]
+# style "preprocessor" as comment, as for HCL lines starting with "#" are comments
+preprocessor=comment
```

Comment:
Hum… sounds like a hack :)  I would suggest giving `[styling=HTML]` and `lexer_filetype=Zephir` a try.  It looks a bit weird with the default colors because for some reasons strings are getting green, but Zephir being basically PHP it has all of `#`, `//` and `/* */` comments builtin.  Just a suggestion, the hack here seems to work.

Reply:
Yeah, Zephir works better than C. Changed it in 3784ce90bf47753fa0e1bc4b7f2bd580d5dcf33b and added a more detailed comment at the top of the file to explain that it is a mix of HTML styling, Zephir/PHP lexing and Terraform ctags parsing :).

I left the colors as is, I think it's okish and with multiple other themes it's actually fine.